net/client_grpc.go

```diff
@@ -324,6 +327,43 @@ func (g *grpcClient) conn(p Peer) (*grpc.ClientConn, error) {
 	return c, err
 }
 
+type httpHandler struct {
+	httpgrpc.HTTPClient
+}
+
+func (h *httpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	req, err := httpgrpcserver.HTTPRequest(r)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	resp, err := h.Handle(r.Context(), req)
+	if err != nil {
+		var ok bool
+		resp, ok = httpgrpc.HTTPResponseFromError(err)
+
+		if !ok {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+	}
+
+	if err := httpgrpcserver.WriteResponse(w, resp); err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+}
+
+func (g *grpcClient) HandleHTTP(p Peer) (http.Handler, error) {
+	conn, err := g.conn(p)
+	if err != nil {
+		return nil, err
+	}
+	client := httpgrpc.NewHTTPClient(conn)
```

Comment:
Could we get this documented ? This is not a usual setup and it'd be useful to know that there is indeed a http API that is fetched from every drand nodes via a custom encoding. That's not going to be easily compatible in other languages for example (not that we care _for now_).

Reply:
where is the right place for that?